modules/logging-forwarding-azure.adoc

```diff
@@ -128,10 +128,10 @@ spec:
        name: my-secret
   pipelines:
    - name: app-pipeline
-    inputRefs:
-    - application
-    outputRefs:
-    - azure-monitor
+     inputRefs:
```

Comment:
Please revert change from 131 to 134.

Reply:
Hello @anpingli ,
Reverted the suggested changes from 131 to 134.

Kindly take a look and provide QE approval for this change.
Thank you.